google-cloud-datastore/lib/google/cloud/datastore/aggregate_query.rb

```diff
@@ -122,6 +122,114 @@ def add_count aggregate_alias: nil
           self
         end
 
+        ##
+        # Adds a sum aggregate.
+        #
+        # @param name [String] The property to sum by.
+        # @param aggregate_alias [String] Alias to refer to the aggregate. Optional
+        #
+        # @return [AggregateQuery] The modified aggregate query object with the added SUM aggregate.
+        #
+        # @example
+        #   require "google/cloud/datastore"
+        #
+        #   datastore = Google::Cloud::Datastore.new
+        #
+        #   query = Google::Cloud::Datastore::Query.new
+        #   query.kind("Task")
+        #        .where("done", "=", false)
+        #
+        #   Create an aggregate query
+        #   aggregate_query = query.aggregate_query
+        #                          .add_sum("score")
+        #
+        #   aggregate_query_results = dataset.run_aggregation aggregate_query
+        #   puts aggregate_query_results.get
+        #
+        # @example Alias an aggregate SUM query
+        #   require "google/cloud/datastore"
+        #
+        #   datastore = Google::Cloud::Datastore.new
+        #
+        #   query = Google::Cloud::Datastore::Query.new
+        #   query.kind("Task")
+        #        .where("done", "=", false)
+        #
+        #   # Create an aggregate query
+        #   aggregate_query = query.aggregate_query
+        #                          .add_sum("score", aggregate_alias: 'total_score')
+        #
+        #   aggregate_query_results = dataset.run_aggregation aggregate_query
+        #   puts aggregate_query_results.get('total_score')
+        #
+        def add_sum name, aggregate_alias: nil
+          aggregate_alias ||= ALIASES[:sum]
```

Comment:
Small nit: I would just create separate constants `DEFAULT_SUM_AGGREGATE_ALIAS`, `DEFAULT_AVG_AGGREGATE_ALIAS` and `DEFAULT_COUNT_AGGREGATE_ALIAS` rather than do a hash lookup each time. I think it's more clear (and simple).

Reply:
Added private constants and used them instead of a hash lookup.